Release/src/utilities/asyncrt_utils.cpp

```diff
@@ -736,6 +736,12 @@ static compute_year_result compute_year(int64_t secondsSince1900)
     int secondsInt = static_cast<int>(secondsLeft - year4 * SecondsIn4Years);
 
     int year1 = secondsInt / SecondsInYear;
+    if (year1 == 4)
+    {
+        // this is the last day in a leap year
```

Comment:
This comment is a little confusing since we are talking about a day, but the variable is year.